src/ripple/protocol/impl/LedgerFormats.cpp

```diff
@@ -141,6 +141,7 @@ LedgerFormats::LedgerFormats()
         {
             {sfAmendments,           soeOPTIONAL},  // Enabled
             {sfMajorities,           soeOPTIONAL},
+            {sfSequence,             soeOPTIONAL},
```

Comment:
Why have a separate `Sequence` field instead of just using the existing logic for `sfPreviousTxnID`/`sfPreviousTxnLgrSeq`?

Reply:
I would prefer those fields too.  It would allow walking the activation change txs.

Reply:
https://github.com/XRPLF/rippled/pull/4751 - IMO this is a better solution since it aligns with the existing ledger structure